.zenodo.json

```diff
@@ -482,21 +559,13 @@
             "affiliation": "1 - Rosseland Centre for Solar Physics, University of Oslo, P.O. Box 1029 Blindern, NO-0315 Oslo, Norway 2 - Institute of Theoretical Astrophysics, University of Oslo, P.O. Box 1029 Blindern, NO-0315 Oslo, Norway "
         },
         {
-            "name": "Yudhik Agrawal",
-            "orcid": "0000-0003-3827-6857",
-            "affiliation": "International Institute of Information Technology, Hyderabad 500 032, India"
-        },
-        {
-            "name": "jamescalixto"
-        },
-        {
-            "name": "yasintoda"
+            "name": "Ambar Mehrotra"
         },
         {
-            "name": "Sophie A. Murray",
-            "orcid": "0000-0002-9378-5315",
-            "affiliation": "Trinity College Dublin, Ireland"
```

Comment:
it probably shouldn't drop people from the list :wink:

Reply:
I guess Sophie hasn't made a commit to the repository. I can easily hard code her in to `update_zenodo.py`, but maybe it would be good to have some more proper way of collecting contributors who haven't contributed via. a commit.

Reply:
Actually, I worked out how to fix this, now anyone in `.zenodo.json` is retained even if they haven't made a commit.